jpa/src/main/resources/META-INF/persistence.xml

```diff
@@ -30,6 +30,7 @@
 	    <jta-data-source>jdbc/DefaultDataSource</jta-data-source>
         <!-- needed by eclipselink to do table creates/drops -->
         <non-jta-data-source>jdbc/DefaultDataSourceNonJTA</non-jta-data-source>
+        <class>org.example.app.Person</class>
```

Comment:
Still curious why this is different btw. Maven & Gradle (and needed just in Gradle).    Is the loose app format any different?

This doesn't need answering now BTW.

Reply:
I will need to revise this again and see if it still fails. Enough things have changed since the last time i tested this, that this may no longer be an issue.

Reply:
This workaround is still required until we figure out what exactly is causing this the Person entity from not being recognized (java.lang.IllegalArgumentException: Object: org.example.app.Person@64235cc0 is not a known Entity type.). We should investigate this separately. I will create an issue for this.